app/shared/src/navigation/HeaderRightButton.js

```diff
@@ -11,14 +11,35 @@ type Props = {|
   onPress?: () => void,
   pressColorAndroid?: string,
   tintColor?: string,
+  onLongPress?: (React.Element<*>) => void,
 |};
 
-export default class HeaderRightButton extends React.PureComponent<Props> {
+type State = {|
+  reference: React.Element<*> | null,
+|};
+
+export default class HeaderRightButton extends React.PureComponent<
+  Props,
+  State,
+> {
+  state = {
+    reference: null,
+  };
   static defaultProps = {
     pressColorAndroid: 'rgba(0, 0, 0, .32)',
     tintColor: '#fff',
   };
 
+  refHandler = (reference: any) => {
```

Comment:
Definitely not `any`. You would discover that the types are wrong - it should be `React.ElementRef` if you are working with references...